tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,471 @@ def test_reads_during_server_pod_restart(self):
+        LOGGER.info("Step 9: Successfully created IAM user with multiple buckets and ran IOs.")
+        LOGGER.info("COMPLETED: Test to verify continuous DELETEs during server pod restart.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-44838")
+    def test_read_write_during_server_pod_restart(self):
+        """
+        Verify READ/WRITEs during server pod restart
+        """
+        LOGGER.info("STARTED: Test to verify READ/WRITE during server pod restart.")
+        LOGGER.info("STEP 1: Perform WRITEs/READs/Verify with variable object sizes")
+        LOGGER.info("Create 2 set of buckets to be used for writes and reads")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        test_prefix_read = 'test-44838-read'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_read, skipcleanup=True,
+                                                    nsamples=5, nclients=5, skipread=True)
+        assert_utils.assert_true(resp[0], resp[1])
+        test_prefix_write = 'test-44838-write'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_write, skipcleanup=True,
+                                                    nsamples=5, nclients=2, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITEs/READs/Verify with variable sizes objects.")
+        num_replica = self.num_replica - 1
+        LOGGER.info("Step 2: Shutdown server pod with replica method and verify cluster & "
+                    "remaining pods status")
+        resp = self.ha_obj.delete_kpod_with_shutdown_methods(
+            master_node_obj=self.node_master_list[0], health_obj=self.hlth_master_list[0],
+            pod_prefix=[const.SERVER_POD_NAME_PREFIX], delete_pod=[self.delete_pod],
+            num_replica=num_replica)
+        # Assert if empty dictionary
+        assert_utils.assert_true(resp[1], "Failed to shutdown/delete server pod")
+        pod_name = list(resp[1].keys())[0]
+        self.set_name = resp[1][pod_name]['deployment_name']
+        self.restore_pod = True
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown server pod %s. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
+        event = threading.Event()  # Event to be used to send intimation of server pod restart
+        LOGGER.info("Step 3: create new buckets and start READ/WRITEs/VERIFY with variable object"
+                    " sizes in background")
+        test_prefix_write = 'test-44838-deg-write'
```

Comment:
why creating another set of buckets now? can't we use same we created in step 1?